bokeh/properties.py

```diff
@@ -175,7 +175,8 @@ def matches(self, new, old):
         except (KeyboardInterrupt, SystemExit):
             raise
         except Exception as e:
-            logger.debug("could not compare %s and %s for property %s (Reason: %s)", new, old, self.name, e)
+            # if we cannot compare (e.g. arrays) just punt return False for match
+            pass
```

Comment:
is there any way to ignore only that numpy error? it's pretty easy to imagine other kinds of bugs being eaten here